docs/cloud/aws/lca1/customer-ssp/index.md

```diff
@@ -0,0 +1,97 @@
+control set.  By using the CFS **LCA1** service, we help you meet some of your security controls.
+Below is a list of features we offer that can help you with your SSP.
+
+```{important}
+Using the CFS **LCA1** service does not automatically make your system compliant.  Our system is a foundation that enables you to be compliant.
+```
+
+**Federated Identity and Access Management (IAM)**
+
+When using the CFS **LCA1** landing zone, role access is integrated with the CU Boulder
+[OIT Identity and Access Management](https://oit.colorado.edu/services/identity-access-management)
+(IAM) offering.  This offering gives you:
+* Single Sign On (SSO) using your [CU Boulder IdentiKey](https://oit.colorado.edu/services/identity-access-management/identikey)
+* [Duo Multi-factor Authentication](https://oit.colorado.edu/services/identity-access-management/duo-multi-factor-authentication)
+* Access management via [Enterprise Access Management (Grouper)](https://oit.colorado.edu/services/identity-access-management/enterprise-access-management)
+
+You have full control over who can access your AWS accounts.
+
+**AWS Root Account User Protection**
+
+CFS secures the [AWS Root Account User](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html)
+for all your **LCA1** accounts. We add MFA to the Root User.  We securely store and rotate the password.
+We have alarms set up to detect any usage of this important and powerful admin user.  Our internal
+procedures ensure that the root user is only used for approved use, requires approval, and is only
+used for very rare cases.  For a list of things that only the root user can do,
+[see this page](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#root-user-tasks).
+In the rare event that any of these actions are needed, [rc-support@colorado.edu](mailto:rc-support@colorado)
+can assist with submitting a ticket with the CFS team.
+
+**Centralized Audit Logging**
+
+When using the CFS **LCA1** landing zone all user, role and service actions are centrally recorded
+and stored with the proper data retention and audit policies.
+This includes actions in the AWS Console, the AWS CLI, and AWS SDKs and APIs.
+For more information, see [AWS CloudTrail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-user-guide.html) documentation.
+This feature allows you to focus on your application logging without the need to build and maintain an extra audit logging layer.
+
+**Managed Network and Firewall**
+
+When using the CFS managed [AWS VPC](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html),
+you have a secure encrypted private subnet connecting back to CU Boulder campus.
+You also get a internet subnet that we manage.  Both subnets are inspected by a firewall.
+Both inbound and outbound traffic are inspected.
+The firewall and campus connection are managed by a partnership with
```

Comment:
Should we link to https://curc--382.org.readthedocs.build/en/382/cloud/aws/lca1/networking.html for more information?

Reply:
Good one.  I added a new paragraph to the bottom of the section.